### PR TITLE
vagrant files to build the development environment

### PR DESCRIPTION
**Pull request by sav**

### CODE REVIEW

**Comment by sav** [comment]
missing:

- test on Linux
- setup a veth
- compile, install and run dome on startup

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1 @@
+* text=auto
```
Comment (marcelstanley) [inline .gitattributes:1]: Perhaps we could move this to a separate commit

**Review by marcelstanley — COMMENTED** [review]
Besides the comment below, I'd also change the commit message to something like:
`add vagrant files to build dev`

Also, as discussed offline, let's keep all providers for now. A later PR should removed them

**Review by marcelstanley — APPROVED** [review]
